src/components/explorer/IndividualMedicalActions.js

```diff
@@ -0,0 +1,324 @@
+                        radiationTherapy={medicalAction.radiation_therapy}
+                        resourcesTuple={resourcesTuple}
+                    />
+                }
+                {medicalAction?.therapeutic_regimen &&
+                    <TherapeuticRegimen
+                        therapeuticRegimen={medicalAction.therapeutic_regimen}
+                        resourcesTuple={resourcesTuple}
+                    />
+                }
+            </Descriptions.Item>
+            <Descriptions.Item label="Adverse Events">
+                { Array.isArray(medicalAction?.adverse_events) ?
+                    medicalAction.adverse_events.map((advEvent, index) =>
+                        <OntologyTerm resourcesTuple={resourcesTuple} term={advEvent} key={index}/>)
+                    : EM_DASH
+                }
+            </Descriptions.Item>
+        </Descriptions>
+    );
+};
+MedicalActionDetails.propTypes = {
+    medicalAction: medicalActionPropTypesShape,
+    resourcesTuple: PropTypes.array,
+};
+
+const adverseEventsCount = (medicalAction) => {
+    return (medicalAction?.adverse_events ?? []).length;
+};
+
+const MedicalActions = ({medicalActions, resourcesTuple, handleMedicalActionClick}) => {
+    const expandedRowRender = useCallback(
+        (medicalAction) => (
+            <MedicalActionDetails
+                medicalAction={medicalAction}
+                resourcesTuple={resourcesTuple}
+            />
+        ), [resourcesTuple],
+    );
+
+    const MEDICAL_ACTIONS_COLUMS = useMemo(() => [
+        {
+            title: "Action Type",
+            key: "action",
+            render: (_, medicalAction) => {
+                return getMedicalActionType(medicalAction).name;
+            },
+            sorter: (a, b) => {
+                const aType = getMedicalActionType(a).type;
+                const bType = getMedicalActionType(b).type;
+                return aType.localeCompare(bType);
+            },
+        },
+        {
+            title: "Treatment Target",
+            key: "target",
+            render: conditionalOntologyRender("treatment_target", resourcesTuple),
+            sorter: ontologyTermSorter("treatment_target"),
+        },
+        {
+            title: "Treatment Intent",
+            key: "intent",
+            render: conditionalOntologyRender("treatment_intent", resourcesTuple),
+            sorter: ontologyTermSorter("treatment_intent"),
+        },
+        {
+            title: "Response To Treatment",
+            key: "response",
+            render: conditionalOntologyRender("response_to_treatment", resourcesTuple),
+            sorter: ontologyTermSorter("response_to_treatment"),
+        },
+        {
+            // Only render count, expand for details
+            title: "Adverse Events",
+            key: "adverse_events",
+            render: (_, medicalAction) => adverseEventsCount(medicalAction),
+            sorter: (a, b) => {
```

Comment:
i think this function doesn't have to be this verbose, since it's pretty clear from the function names what is going on - could just be

```js
(a, b) => adverseEventsCount(a) - adverseEventsCount(b),
```